### PR TITLE
Persist generated private key in Keychain DB

### PR DESCRIPTION
**Pull request by sirn**
Issue/Task Number: #689

### CODE REVIEW

File: apps/keychain/config/prod.exs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+config :keychain, Keychain.Repo,
+  adapter: Ecto.Adapters.Postgres,
+  url: {:system, "DATABASE_URL", "postgres://localhost/ewallet_prod"},
```
Comment (unnawut) [inline apps/keychain/config/prod.exs]:
```suggestion
  url: {:system, "DATABASE_URL", "postgres://localhost/ewallet"},
```

File: apps/keychain/test/keychain/key_test.exs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    test "inserts a new private key" do
+      assert Repo.all(Key) == []
+      {:ok, key} = Key.insert_private_key("key-1", "private-key-1")
+      assert Repo.all(Key) == [key]
+    end
+
+    test "raises if wallet id already exists" do
+      key_1 = insert(:key)
+
+      assert_raise ConstraintError, fn ->
```
Comment (unnawut) [inline apps/keychain/test/keychain/key_test.exs]: Should this return `{:error, changeset}` instead?

File: apps/blockchain_eth/lib/blockchain_eth/worker.ex [hunk 3]
```diff
@@ -73,7 +74,10 @@ defmodule BlockchainEth.Worker do
       |> Keccak.kec()
       |> Base.encode16(case: :lower)
 
-    {:reply, {:ok, "0x#{account_encoded}", pkey_encoded}, reg}
+    wallet_id = "0x#{account_encoded}"
+
+    {:ok, _} = Key.insert_private_key(wallet_id, pkey_encoded)
```
Comment (T-Dnzt) [inline apps/blockchain_eth/lib/blockchain_eth/worker.ex]: This should be storing the private key, not the public one 😅

File: apps/keychain/test/keychain/key_test.exs (hunk 2 above)
Comment (sirn) [reply apps/keychain/test/keychain/key_test.exs]: I don't think so, this is a "should never happen" scenario and I think it's better to be hard fail than a soft fail.

**Pull request merged**